sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 5.8.0b5 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.8.0b5 (2022-07-19)
 
 ### Bugs Fixed
 
+- Bug fixes that would prevent token refresh at regular intervals
+- Pass in the proper kwarg, so that debug levels will be output when requested
```

Comment:
for conformity with rest of changelog, maybe rewording like this?
```suggestion
- Fixed bug that prevented token refresh at regular intervals.
- Fixed bug that was improperly passing the debug keyword argument, so that network trace debug logs are output when requested.
```
for second bullet, I think this was the network_trace kwarg, right?

Reply:
right it was the network_trace